components/User/UserProfile/Integrations.js

```diff
@@ -338,15 +413,15 @@ class Integrations extends React.Component {
                           <div className="two wide column">
                             {
                               (github) ? (
-                                <button className={github_disable_classes} name="github" onClick={this.handleDisable.bind(this)} >Disable</button>
+                                <button className={github_disable_classes} name="github" onClick={this.handleDisable.bind(this)} ><FormattedMessage id='Integration.disableGithub' defaultMessage='Disable'/></button>
                               ) : (
-                                <button className={github_enable_classes} name="github" onClick={this.handleEnable.bind(this)} >Enable</button>
+                                <button className={github_enable_classes} name="github" onClick={this.handleEnable.bind(this)} ><FormattedMessage id='Integration.enableGithub' defaultMessage='Enable'/></button>
```

Comment:
Somehow this breaks the event transmitted to the function. On master the button is working.